monitoring/mixin/Makefile

```diff
@@ -0,0 +1,23 @@
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+
+default: build
+
```

Comment:
Maybe add a target to download these tools, similar to our "deps" target in skipper Makefile